lte/gateway/c/oai/include/pgw_config.h

```diff
@@ -115,7 +115,7 @@ typedef struct pgw_config_s {
 
   bool force_push_pco;
   uint16_t ue_mtu;
-  bool relay_enabled;
+  bool hss_relay_enabled;
```

Comment:
Remove this as relay_enabled is not used in the SPGW task any more. All session messages are sent to SessionD.